redis/commands/json/commands.py

```diff
@@ -253,6 +253,26 @@ def set(
             pieces.append("XX")
         return self.execute_command("JSON.SET", *pieces)
 
+    def mset(self, name: str, path: str, obj: JsonType, *items) -> Optional[str]:
```

Comment:
IMHO the interface shoudl be changed. It's unclear that these are collections of name/path/str objects

Reply:
@shacharPash We should allow passing list instead:

```python
# Interface:
def mset(self, triplets: List[Tuple[str, str, int]]) -> Optional[str]:
# ...

# Usage:
json.mset([("user:1", "$.company", "Redis"), ("user:2", "$.company", "Redis")])
```